src/locales/en/index.js

```diff
@@ -140,7 +140,7 @@ export default {
     francisca: "Francisca",
     king: "King",
     jack: "Jack",
-    ethel: "Ezerbald",
+    ethel: "Ezerbald", // TODO: ask later
```

Comment:
[Question]

I cited the name from [King Ethelbald](https://en.wikipedia.org/wiki/%C3%86thelbald,_King_of_Wessex).

Is Ezerbald more natural ?

Both of them are ok for me.

Reply:
Ethelbald sounds better.  I think he's mentioned by name in a few parts of the text, I'd make sure to do a search for "Ezerbald" and change them all.

Reply:
Thank you. I replaced all of them.